### ISSUE TITLE
Only version 2.X of the docker-compose format

### ISSUE THREAD

**Issue by brainiac91** [fabric8io/docker-maven-plugin#1606]
I'm trying to parse a compose file that has features from compose version 3.x such as:
    depends_on:
      create_database:
        condition: service_completed_successfully

However. I'm prompted with the following when running "mvn docker:start":

Execution default-cli of goal io.fabric8:docker-maven-plugin:0.40.2:start failed: Only version 2.x of the docker-compose format is supported for C:\path\to\src\main\resources\compose.yaml -> [Help 1]

Is there any workaround for this? Will this be supported in future updates?

* docker-maven-plugin version : 0.40.2
Docker version 20.10.13
Apache Maven 3.8.5
Java version: 11.0.14.1

**Comment by rhuss** [comment]
Nope, except when you downgrade to docker-compose version 2. I have no bandwidth to work on a compose 3 update, but let me know if you would like to contribute it to dmp.

**Comment by poikilotherm** [comment]
Actually this `depends_on` syntax has been introduced in compose v2.1, not v3. There is issue #888, so maybe this one here can be closed as a duplicate?
